content/release-10-15-0/devices-10-15-0-bundle/10-15-0-386.md

```diff
@@ -0,0 +1,59 @@
+---
+weight: 18
+title: Release 10.15.0.386
+layout: bundle
+---
+
+<!--10.15.0.378 - 10.15.0.386-->
+
+### Fixes
+
+<div><table ><colgroup>
+<col style="width: 15%;"><col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Component</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Device Management</td>
+<td>In the SmartREST templates editor, issues with the presentation of the <b>External ID type</b> field for inventory POST messages have been fixed. Under <b>CSV preview</b>, the generated "Template creation CSV" has been adjusted to include the ID/externalId/externalIdType values.</td>
+<td>DM-2093</td>
+</tr>
+
+<tr>
+<td>Device Management</td>
+<td>In the device grid, issues with applying filters have been fixed.</td>
+<td>DM-2321</td>
+</tr>
+
+<tr>
+<td>LWM2M</td>
+<td>Bulk device registrations and other operations being executed on the LWM2M connector device now show the status FAILED if a problem occurs. Prior to this change, partial failures were reported as SUCCESSFUL.</td>
+<td>DM-1951</td>
+</tr>
+
+<tr>
+<td>LWM2M</td>
+<td>The LWM2M agent now ignores trailing commas at the end of object links in the registration request of a LWM2M client.</td>
+<td>DM-2342</td>
+</tr>
+
+<tr>
+<td>LWM2M</td>
+<td>In certain cases, the registration couldn't be associated to a LWM2M 1.1 SEND request, leading to a 4.04 CoAP response. This is now fixed.</td>
```

Comment:
```suggestion
<td>In certain cases, the registration couldn't be associated to an LWM2M 1.1 SEND request, leading to a 4.04 CoAP response. This is now fixed.</td>
```

is this a 404? it might be an erroneous "."